### DISCUSSION TITLE
Friday night funkin kade engine 1.6.1 Where are the files that reposition the KadeEngineLogoBumpin

### DISCUSSION THREAD

**Discussion by Reyororo** [KadeArchive/Kade-Engine#1642] [Q&A] [answered]
Please tell me where is the file where I can change the position of KadeEngineLogoBumpin

**Comment by raf13lol** [comment, marked as answer]
at what point? the loading screen or the title screen?

**Reply by Reyororo** [reply]
> どの時点で？ロード画面またはタイトル画面？

title screen

**Reply by raf13lol** [reply]
In TitleState.hx
